jbdevstudio/multiple/jbdevstudio-multiple.target

```diff
@@ -122,17 +122,17 @@
 
     <!-- Eclipse -->
     <location includeAllPlatforms="false" includeMode="slicer" type="InstallableUnit" includeSource="true">
-      <repository location="http://download.jboss.org/jbosstools/updates/requirements/mars/201505081000-M7/"/>
+      <repository location="http://download.jboss.org/jbosstools/updates/requirements/mars/201505291000-RC2/"/>
 
       <!-- m2e -->
-      <unit id="org.eclipse.m2e.feature.feature.group" version="1.6.0.20150505-1732"/>
+      <unit id="org.eclipse.m2e.feature.feature.group" version="1.6.0.20150526-2032"/>
       <!-- Required for m2e tests -->
-      <unit id="org.eclipse.m2e.tests.common" version="1.6.0.20150505-1732"/>
+      <unit id="org.eclipse.m2e.tests.common" version="1.6.0.20150526-2032"/>
       <!-- m2e-wtp -->
-      <unit id="org.eclipse.m2e.wtp.feature.feature.group" version="1.2.0.20150312-1808"/>
-      <unit id="org.eclipse.m2e.wtp.jaxrs.feature.feature.group" version="1.2.0.20150312-1808"/>
-      <unit id="org.eclipse.m2e.wtp.jpa.feature.feature.group" version="1.2.0.e43-20150312-1809"/>
-      <unit id="org.eclipse.m2e.wtp.jsf.feature.feature.group" version="1.2.0.20150312-1808"/>
+      <unit id="org.eclipse.m2e.wtp.feature.feature.group" version="1.2.0.20150506-1407"/>
```

Comment:
RC2 repo for m2e-wtp is http://download.eclipse.org/m2e-wtp/milestones/mars/1.2.0/1.2.0.20150526-1758/

So all plugins version is 1.2.0.20150526-1758 and org.eclipse.m2e.wtp.jpa.feature.feature.group is 1.2.0.e43-20150526-1800